### PR TITLE
Feature delices updates

### PR DESCRIPTION
**Pull request by abates**

### CODE REVIEW

File: nautobot_design_builder/choices.py [hunk 1]
```diff
@@ -15,17 +15,3 @@ class DesignInstanceStatusChoices(ChoiceSet):
         (DISABLED, DISABLED),
         (DECOMMISSIONED, DECOMMISSIONED),
     )
-
-
-class DesignInstanceLiveStateChoices(ChoiceSet):
-    """Status choices for Live State Designs Instance."""
-
-    DEPLOYED = "Deployed"
```
Comment (chadell) [inline nautobot_design_builder/choices.py:23]: to capture the same meaning, could we add these choices to the other `Status` field?

File: nautobot_design_builder/signals.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def create_design_instance_statuses(**kwargs):
         "Pending": ColorChoices.COLOR_ORANGE,
         "Rolled back": ColorChoices.COLOR_RED,
     }
-    for _, status_name in chain(choices.DesignInstanceStatusChoices, choices.DesignInstanceLiveStateChoices):
+    for _, status_name in chain(choices.DeploymentStatusChoices):
```
Comment (chadell) [inline nautobot_design_builder/signals.py:52]: you have to create the "Uknown" status that is defined in the choices in the previous mapping. Also, remove the non used ones

File: nautobot_design_builder/choices.py (hunk 1 above)
Comment (abates) [reply nautobot_design_builder/choices.py:23]: Base on our conversation earlier in the week I think I remember that you said we would leave this up to the end user to decide which statuses to have. Am I remembering correctly, or do you still want me to add these back in?

**Comment by abates** [comment]
@chadell ready for your review.

File: nautobot_design_builder/choices.py (hunk 1 above)
Comment (chadell) [reply nautobot_design_builder/choices.py:23]: yes, you are right. Let's keep some basic ones, and leave the user to extend

**Review by chadell — APPROVED** [review]
LGTM
🎉 thanks @abates

**Pull request merged**